static/app/views/nav/primary/components.tsx

```diff
@@ -313,10 +459,28 @@ export const NavButton = styled('button', {
   ${baseNavItemStyles}
 `;
 
-export const SidebarItemUnreadIndicator = styled('span')`
+interface NavButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
+  isMobile: boolean;
+}
+
+export const NavButton = styled((p: NavButtonProps) => {
+  const theme = useTheme();
+  if (theme.isChonk) {
+    return (
+      <ChonkNavButton
+        {...p}
+        aria-label={p['aria-label'] ?? ''}
+        size={p.isMobile ? 'zero' : undefined}
+      />
+    );
+  }
+  return <StyledNavButton {...p} />;
+})``;
+
+export const SidebarItemUnreadIndicator = styled('span')<{isMobile: boolean}>`
   position: absolute;
-  top: calc(50% - 12px);
-  left: calc(50% + 12px);
+  top: ${p => (p.isMobile ? `8px` : `calc(50% - 5px)`)};
+  left: ${p => (p.isMobile ? '32px' : `calc(50% + 12px)`)};
```

Comment:
those numbers look pretty magical to me 😅

Reply:
Egh, unfortunately not trivial as the buttons are a lot different size in chonk vs the old size